modules/fft/float/dsps_fft2r_fc32_ansi.c

```diff
@@ -40,7 +40,7 @@ unsigned short reverse(unsigned short x, unsigned short N, int order);
 esp_err_t dsps_fft2r_init_fc32(float *fft_table_buff, int table_size)
 {
     esp_err_t result = ESP_OK;
-    if (dsps_fft2r_initialized != 0) {
+    if (dsps_fft2r_initialized != 0 && fft_table_buff == NULL) {
```

Comment:
The logic of the dsps_fft2r_init_fc32() is that it have to be used only once. We can't change the pointer. If you whant to do this, you have to call dsps_fft2r_deinit_fc32() first, and than call dsps_fft2r_init_fc32() again.